homework-assignments-1/task_4.ts

```diff
@@ -0,0 +1,170 @@
+class Product {
```

Comment:
Please make all class member as private due to security best practices. Your should communicate class members only via getter and setters.